di-ipv-core-stub/src/main/java/uk/gov/di/ipv/stub/core/config/uatuser/IdentityMapper.java

```diff
@@ -114,6 +142,11 @@ public List<QuestionAndAnswer> mapToQuestionAnswers(
     }
 
     public Identity mapFormToIdentity(Identity identityOnRecord, QueryParamsMap userData) {
+        List<UKAddress> addresses = new ArrayList<>();
+
+        LocalDate vf = getLocalDate(userData, "validFrom-year", "validFrom-month", "validFrom-day");
```

Comment:
rename the variables to make their meaning/purpose more obvious:
e.g. primaryAddressValidFrom primaryAddressValidUntil